recipes/sciris/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "sciris" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sciris-{{ version }}.tar.gz
+  sha256: 46b81d117cef78348667c2cf727d52ed67a4b79c2b6a0cb19c133121b5881396
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib-base
+    - numpy
+    - pandas
+    - openpyxl
+    - xlsxwriter
+    - psutil
+    - dill
+    - zstandard
+    - multiprocess
+    - jsonpickle
+    - pyyaml
+    - packaging
+    - gitpython
+    - jellyfish
+    - tqdm
+    - line_profiler
+    - memory_profiler
+    - colorama  # [win]
```

You can drop the Windows selector and make this noarch.
```suggestion
    - colorama
```